src/main/java/lrgs/lrgsmon/LrgsMonCmdLineArgs.java

```diff
@@ -180,20 +180,13 @@ public void parseArgs(String args[])
 		if (fn != null && fn.length() > 0)
 		{
 			String procname = Logger.instance().getProcName();
```

Comment:
This pattern of setting up a FileLogger is also in LqmCmdLineArgs, and DrgsCmdLineArgs,  ...    perhaps a base or utility class could take this on.

Reply:
Hmm, that's a good idea. The duplication was bugging me but wasn't sure what to do.
I'll see about a utility class.